src/site/stages/build/process-cms-exports/schemas/transformed/node-page.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
```

Comment:
This file was accidentally brought over from @cvalarida's PR lol

Reply:
:see_no_evil: It'll be changed; prolly should nuke it if you can.